queen-infra-db/src/main/resources/db/changelog/610_cipher_data.sql

```diff
@@ -0,0 +1,19 @@
+--liquibase formatted sql
+
+--changeset davdarras:610-0
+ALTER TABLE data add column encrypted integer;
+UPDATE data set encrypted=0;
+
+--changeset davdarras:610-1 context:ciphered-data
```

Comment:
could add a "true comment" here highlighting context sensitive behavior option `context`
=> liquibase function is "hidden" in comment and can be easily missed